packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -0,0 +1,3 @@
+from pkgpanda.util import load_json
+
+dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
Should we name this differently, so it doesn't conflict with the existing `dcos_config` parameter in the dcos-enterprise downstream?